src/components/ProductImages/ProductImages.css

```diff
@@ -0,0 +1,4 @@
+
+.componentHeight {
+    height: 400px;
+}
```

Comment:
Add a new line at the end to get rid of this

<img width="72" alt="image" src="https://github.com/damdiana/beauty-app/assets/11327761/90b0796e-2e9f-4f04-b99f-6f85c681a7c7">